lib/workload/stateless/stacks/filemanager/deploy/stack.ts

```diff
@@ -141,20 +137,21 @@ export class Filemanager extends Stack {
       ...props,
     });
 
-    const ApiGateway = new ApiGatewayConstruct(this, 'ApiGateway', {
-      region: this.region,
-      apiName: 'FileManager',
-      customDomainNamePrefix: 'file',
-      ...props,
-    });
+    const ApiGateway = new ApiGatewayConstruct(this, 'ApiGateway', props.apiGatewayCognitoProps);
     const httpApi = ApiGateway.httpApi;
 
     const apiIntegration = new HttpLambdaIntegration('ApiIntegration', apiLambda.function);
 
-    new HttpRoute(this, 'HttpRoute', {
+    new HttpRoute(this, 'GetHttpRoute', {
+      httpApi: httpApi,
+      integration: apiIntegration,
+      routeKey: HttpRouteKey.with('/{proxy+}', HttpMethod.GET),
+    });
+
```

Comment:
@mmalenic Wanted to confirm from the [FM docs](https://github.com/umccr/orcabus/blob/main/lib/workload/stateless/stacks/filemanager/docs/API_GUIDE.md), FM only used GET and PATCH method?

Reply:
There's a `POST` route as well on `/api/v1/ingest` which manually ingests data from an SQS queue. It isn't being used right now, but could you also please add it here?

Reply:
Resolved - ccf38b2de940bba17076993616dade51fa375d78